### ISSUE TITLE
Included files flushed

### ISSUE THREAD

**Issue by lchandelier** [FrankFang/gulp-html-extend#24]
Hi,
Thanks for your plugin.
I have a strange issue. I use html files like header.html, footer.html that I include in my main template files such as home.html, contact.html,.... But sometimes, when gulp compiles, these files (header/footer) are flushed and I have to go back in my code history to get it back.

I haven't noticed any trigger that could do this and there are no errors in the console this but it's really annoying. 
Can you please take a look?

**Comment by FrankFang** [comment]
Can you upload your code to gist / github.

The plug never flushes anything. It's strange  🙁

**Comment by lchandelier** [comment]
Of course, you can see how I use it here [https://github.com/mchandelier/korat/tree/master/base_sprites](url)

I use the tasks extend and extend_common in the gulpfile.
